docs/guides/admin/docs/upgrade.md

```diff
@@ -33,7 +33,7 @@ support to be embedded in applications. Since the Elasticsearch client was updat
 external Elasticsearch node of the same version to be present. This means, that all Opencast adopters now have to run
 Elasticsearch.
 
-Please check [the documentation](modules/searchindex/elasticsearch.md) for information about how to setup an external node.
+Please check [the documentation](configuration/elasticsearch.md) for information about how to setup an external node.
```

Comment:
Since the docs don't list these instructions, should we reference the installation instructions instead?
```suggestion
Please check the installation guides for instructions about how to setup Elasticsearch.
```

Reply:
I rewrote this piece, let me know what you think.